_posts/2018-01-04-utah-sgid-statewide-roads-data-layer-updates-01042018.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+status: publish
+published: true
+title: Utah SGID Statewide Roads Data Layer Updates 01/04/2018
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2018-01-04
```

Comment:
I would have thought this needed a time stamp but I guess no? Ill check the preview to see if it built properly.

Reply:
cool. this seems fine.